### PR TITLE
Updates with latest boilerplate code from nf-core create

### PR DESCRIPTION
**Pull request by aunderwo**
## PR checklist
 - [x] This comment contains a description of changes (with reason)
Pulling latest code from dev repo

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,4 +21,5 @@ RUN wget https://download.asperasoft.com/download/sw/connect/3.8.1/ibm-aspera-co
 
 RUN cd /root; git clone https://github.com/enasequence/enaBrowserTools.git; mv enaBrowserTools /enaBrowserTools
```
Comment (apeltzer) [inline Dockerfile]:
```suggestion

```

File: Dockerfile [hunk 2]
```diff
@@ -21,4 +21,5 @@ RUN wget https://download.asperasoft.com/download/sw/connect/3.8.1/ibm-aspera-co
...
 
-ENV PATH /enaBrowserTools/python3:/opt/conda/envs/nf-core-bactmap-1.0dev/bin:$PATH
\ No newline at end of file
+ENV PATH /opt/conda/envs/nf-core-bactmap-1.0dev/bin:$PATH
+ENV PATH /enaBrowserTools/python3:$PATH
```
Comment (apeltzer) [inline Dockerfile]:
```suggestion
```

Comment (apeltzer) [reply Dockerfile]: There is a bioconda package for this: see https://bioconda.github.io/recipes/enabrowsertools/README.html

Comment (aunderwo) [reply Dockerfile]: Thanks @apeltzer, good call. I've pushed this update

**Pull request merged**